src/styles/styles.scss

```diff
@@ -2,3 +2,17 @@
 @import 'tailwindcss/base';
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
+
+@font-face {
+  font-family: IBMPlexSans-Regular;
+  src:
```

Comment:
It's better to make these 2 files as 1 font family, but change the weight
Example
@font-face {
  font-display: swap;
  font-family: IBMPlexSans;
  font-style: normal;
  font-weight: 400;
  src: url('../assets/fonts/IBMPlexSans-Regular.ttf') format('truetype');
}

@font-face {
  font-display: swap;
  font-family: IBMPlexSans;
  font-style: normal;
  font-weight: 600;
  src: url('../assets/fonts/IBMPlexSans-SemiBold') format('truetype');
}